CMakeLists.txt

```diff
@@ -253,12 +253,12 @@ endif()
 # 'DCMQICPackOptions.cmake' and 'dcmqiVersionConfigure.h'
 #
 
-#message("---------")
+message(STATUS "-------------------------------------------------------")
 message(STATUS "Targets '${PkModeling_TARGETS}'")
-#message("---------")
+message(STATUS "-------------------------------------------------------")
 
 include(pkmodelingConfigureVersionTarget)
-add_dependencies(pkmodeling pkmodelingConfigureVersion)
+add_dependencies(PkModeling PkModelingConfigureVersion)
```

Comment:
strange - if I change this line in my tree as you did, it does not help, and still complains about PkModeling as an unknown target. Did you change anything else that didn't get committed?